lua/dap.lua

```diff
@@ -15,6 +15,18 @@ local bp_conditions = {}
 local last_run = nil
 
 M.repl = repl
+M.custom_event_handlers = setmetatable({}, {
+  __index = function(tbl, key)
+    rawset(tbl, key, {})
+    return rawget(tbl, key)
+  end
+})
+M.custom_response_handlers = setmetatable({}, {
+  __index = function(tbl, key)
+    rawset(tbl, key, {})
+    return rawget(tbl, key)
+  end
+})
```

Comment:
Could also be one table. Don't know whether it's reasonable to separate between events and request responses.

Reply:
I'm also not sure, but given that the spec distinguishes between events and responses I think it doesn't hurt to make the distinction explicit here too. Could help avoid ambiguity in the future. So :+1: for separating it.